From 59d3cfa1fee7649378d95273340cc52371a68f6e Mon Sep 17 00:00:00 2001
From: Huang Weiyi <weiyi.huang@gmail.com>
Date: Thu, 8 Apr 2010 19:49:45 +0800
Subject: [PATCH 3/8] Staging: rtl8192u: remove unused #include <linux/version.h>

Remove unused #include <linux/version.h>('s) in
  drivers/staging/rtl8192u/ieee80211/ieee80211_wx.c

Signed-off-by: Huang Weiyi <weiyi.huang@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/rtl8192u/ieee80211/ieee80211_wx.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

--- a/drivers/staging/rtl8192u/ieee80211/ieee80211_wx.c
+++ b/drivers/staging/rtl8192u/ieee80211/ieee80211_wx.c
@@ -30,7 +30,6 @@
 
 ******************************************************************************/
 #include <linux/wireless.h>
-#include <linux/version.h>
 #include <linux/kmod.h>
 #include <linux/slab.h>
 #include <linux/module.h>
